presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -99,6 +99,14 @@ String
 
     Example type definitions: ``varchar``, ``varchar(20)``
 
+    Data insertion in VARCHAR supports usage of unicode literals with the prefix
+    ``U&``, like ``U&`some unicode encoded literal data```. For example, you can
+    use ``U&```1F600``` for a grinning face emoji.
+
+    Binary data insertion is supported with the ``X`` prefix - ``X`some binary
+    encoded data```. The binary data has to use hexadecimal format. For example,
+    the binary form of ``eh?`` is ``X`65683f```
```

Comment:
Nit: use uppercase for hex to be consistent with other usages

Reply:
Done